src/pkg/backup/backup.go

```diff
@@ -95,6 +97,8 @@ type Printable struct {
 	Status     string              `json:"status"`
 	Version    string              `json:"version"`
 	Selectors  selectors.Printable `json:"selectors"`
+	Size       int64               `json:"size"`
+	StoredSize int64               `json:"storedSize"`
```

Comment:
I'm not sure we want/need these displayed on the tabular output for the list.  Our design for that display is more directed to "help the user identify the backup they need" than "show the user all the backup metadata", and extra columns can make the former identification less clear.  Users who need access to metadata will be able to retrieve it with a `--json` output, and sdk users, like you noted, already have access within the struct.

I realize that kind-of makes this a non-issue.  Maybe it'd be good to connect with some sdk consumers and make sure there wasn't a miscommunication about what data needs to be provided.  The ask my extend beyond the sizes we already store.